openglove/resources/settings/default.vrsettings

```diff
@@ -49,16 +49,23 @@
     "left_port": "\\\\.\\COM4",
```

Comment:
Are we planning on making this the default encoding style?

Reply:
If that's acceptable, yes.

Reply:
sure 🚀